repos/system_upgrade/el7toel8/libraries/overlaygen.py

```diff
@@ -17,7 +17,7 @@ def _ensure_enough_diskimage_space(space_needed, directory):
 
 
 def _get_mountpoints(storage_info):
-    return list(set([entry.fs_file for entry in storage_info.fstab if os.path.isdir(entry.fs_file)]))
+    return list({[entry.fs_file for entry in storage_info.fstab if os.path.isdir(entry.fs_file)]})
```

Comment:
Man, really, this is mistake:
```
>>> {[i for i in range(5)]}
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
```

Same on Py2 & Py3. Where you get this? Original solution was more readable.

Reply:
@fellipeh It's not the proper way to write a set comprehension

Reply:
```suggestion
    return list({entry.fs_file for entry in storage_info.fstab if os.path.isdir(entry.fs_file)})
```
To be clear, this could resolve your problem, but `list(set([...`.